articles/users/search.md

```diff
@@ -302,7 +306,84 @@ When you create your job, you'll need to provide:
 }
 ```
 
-Once you've created your job to export your users, you can check on its status using the [Get a Job endpoint](/api/management/v2#!/Jobs/get_jobs_by_id). You'll need to provide the ID of the job (which you received in the response when creating the job) -- if you're using the sample request below, replace the placeholder `YOUR_JOB_ID` with the value of the ID.
+### Export metadata
+
+If you export in CSV and include metadata information, you have to specify each field that you want exported. 
+
+For example, for metadata structured like this:
+
+```json
+{
+  "consent": {
+      "given": true,
+      "date": "01/23/2018",
+      "text_details": "some-url"
+  }
+}
+```
+
+The export request (for all three fields) will be as follows.
+
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/api/v2/jobs/users-exports",
+  "httpVersion": "HTTP/1.1",
+  "cookies": [],
+  "headers": [{
+    "name": "Authorization",
+    "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+  },
+  {
+    "name": "Content-Type",
+    "value": "application/json"
+  }],
+  "queryString": [],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{\"connection_id\": \"YOUR_CONNECTION_ID\", \"format\": \"csv\", \"limit\": 5, \"fields\": [{\"name\": \"email\"}, {\"name\": \"user_metadata.consent.given\"}, {\"name\": \"user_metadata.consent.date\"}, {\"name\": \"user_metadata.consent.text_details\"}]}" 
+    },
+  "headersSize": -1,
+  "bodySize": -1,
+  "comment": ""
+}
+```
+
+If you export in JSON, then you need only provide the root property, and not each individual inner property.
```

Comment:
If you export your user profiles in JSON format, you only need to provide the root property name; you do not need to name each individual inner property since they will be included automatically.